e3sm_to_cmip/runner.py

```diff
@@ -694,15 +696,16 @@ def _run_parallel(self) -> Literal[True]:
                         vars_to_filepaths,
                         self.tables_path,
                         self.new_metadata_path,
+                        self.cmor_log_dir,
```

Comment:
MPAS variable handler `.py` modules now accept `cmor_log_dir` parameter to save their logs their too.